internal/snmp/translate_test.go

```diff
@@ -87,3 +89,54 @@ func TestTrapLookupFail(t *testing.T) {
 		})
 	}
 }
+
+type TestingMibLoader struct {
+	folders []string
+	files   []string
+}
+
+func (t *TestingMibLoader) appendPath(path string) {
+	t.folders = append(t.folders, path)
+}
+
+func (t *TestingMibLoader) loadModule(path string) error {
+	t.files = append(t.files, path)
+	return nil
+}
+func TestFolderLookup(t *testing.T) {
+	paths := [][]string{
+		{"testdata"},
+		{"testdata", "loadMibsFromPath"},
+		{"testdata", "loadMibsFromPath", "linkTarget"},
+		{"testdata", "loadMibsFromPath", "root"},
+		{"testdata", "mibs"},
+	}
```

Comment:
This test is currently kind of halfway written as a table test. There's only one entry in the table so there's not much benefit to having a table. If we're never going to have more than one entry, we should probably remove the table code.

`paths` conceptually belongs with `name` and `files` in the table but it's outside.

We either need to remove the table code or move `paths` inside the table. It should replace `folders` in the table.